removal_from_ansible.rst

```diff
@@ -165,3 +165,58 @@ Re-adding collection to Ansible
 -------------------------------
 
 There is no simplified process. Once the collection has been removed from Ansible Y.0.0, it needs to go through the full inclusion process to be re-added to the Ansible package. Exceptions are only possible if the Steering Committee votes on them. The Steering Committee can approve or deny a fast re-entry without going through the full review process.
+
+Collections not satisfying the Collection Requirements
+======================================================
+
+A collection can be removed from the package if it violates one or more of the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ without making enough effort to mitigate these violations.
+
+This section is not applicable to cases of broken or unmaintained collections. Instead, see the corresponding paragraphs of this document.
+
+Identifying and removing a collection
+-------------------------------------
+
+Conditions for removal
+~~~~~~~~~~~~~~~~~~~~~~
+
+#. A collection violates one or more of the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
+#. Collection maintainers have not fixed the violations and have not released a fixed version of the collection within the time period established by this document.
+
+Process
+~~~~~~~
+
+#. A community person who found a violation creates an issue in the collection's repository. The issue contains:
+
+  * References to corresponding `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ the collection violates.
+  * Actions collection maintainers need to do to make the collection satisfying the requirements.
```

Comment:
This recommends that community members create an community topic if they're unsure whether something constitutes a violation. I think it would also make sense to create a tracking issue for collection violations that people can mention when filling issues so that Github links them. Doing so will allow us to track the situation and identify patterns without making the process overly burdensome. I'm not super attached to the tracking issue idea, but I think it's a good middle ground.


```suggestion
#. Any community member who finds a collection that violates one or more of the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ may file an issue against said collection's repository. If the reporter is unsure whether something constitutes a violation or believes that the apparently violated guideline is unclear, they should consult with the steering committee by filing a `community topic <https://github.com/ansible-community/community-topics/issues>`_ before proceeding.
#. The issue filed against the collection's repository should include the following information:

  * References to the corresponding `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ the collection violates
  * Actions collection maintainers need to do to make the collection comply with the requirements
  * A link the the Collection Requirements violations tracking issue (https://github.com/ansible-community/community-topics/issues/PLACEHOLDER)

```

Reply:
@gotmax23 

> A link the the Collection Requirements violations tracking issue (https://github.com/ansible-community/community-topics/issues/PLACEHOLDER)

I think this should be "to the" instead of "the the", shouldn't it?

Reply:
Indeed. I've fixed it.

Reply:
I like the proposed wording, much better now, thanks! Committed.
I added periods and changed the link in the last bullet point. PTAL

Reply:
The idea was that we would create a single tracking issue so people could add something like `Relates: github.com/ansible-community/community-topics/issue/12345` to the end of the issues filled against collections. Then, every issue that included that link would show up on the tracking issue's page. PLACEHOLDER was meant to be replaced with the ID of that new issue.

In Fedora, we liberally use Bugzilla's support for tracking bugs, but not everyone uses that, so I should've explained myself better :smile:. This is me trying to replicate that with Github's functionality. I wanted find an easy way for the SC to keep track of these violations without making people create issues against two different repositories. I see the issue linking approach might be confusing, won't send notifications, and won't work for collections outside of Github, so maybe we should scrap it...

Reply:
> The idea was that we would create a single tracking issue so people could add something like Relates: github.com/ansible-community/community-topics/issue/12345 to the end of the issues filled against collections. Then, every issue that included that link would show up on the tracking issue's page. PLACEHOLDER was meant to be replaced with the ID of that new issue.

@gotmax23 thanks for the clarifications. If I understand correctly, it's gonna be like a permanent issue, right? If yes, it works for me (regarding the second paragraph: we could try and change it later if needed).
How about @mariolenz @felixfontein ?
If everyone is happy with the tracking issue, let's change it when the issue is created (i believe we should vote as usual before creating the issue)?

Reply:
> If I understand correctly, it's gonna be like a permanent issue, right?

Exactly.

> (i believe we should vote as usual before creating the issue)?

I think it can be one vote. If the motion passes, we can then change the link and create the issue.

Reply:
> I think it can be one vote. If the motion passes, we can then change the link and create the issue.

SGTM